prime-router/metadata/HL7/catchall/hl7/segments/IN1/Coverage.yml

```diff
@@ -13,4 +13,36 @@ extension:
   expressions:
     - expressionType: resource
       valueOf: datatypes/AUI/AUIExtension
-      specs: IN1.14
\ No newline at end of file
+      specs: IN1.14
+    - expressionType: resource
```

Comment:
I think these values will move to `IN1Extension` but that doesn't have to be done now.

Reply:
I didn't get to this before the codeowner review was left. In the interest of not making reviewers revise an already approved PR I left a comment to have this be done as part of the main IN1 work. https://github.com/CDCgov/prime-reportstream/issues/15500